launchpad-starter-next-app/src/services/container.ts

```diff
@@ -1,71 +1,50 @@
-import { TappdClient } from "@phala/dstack-sdk";
-import { privateKeyToAccount } from "viem/accounts";
-import { keccak256 } from "viem";
 import { exec } from "child_process";
 import { promisify } from "util";
+import path from "path";
 
 const execAsync = promisify(exec);
 
 const TEE_SERVER_PORT = process.env.TEE_SERVER_PORT || 8090;
-const TEE_SERVER_URL = process.env.TEE_SERVER_URL || "http://localhost:8090";
+const IS_DEV = process.env.NODE_ENV === "development";
 
 export class ContainerManager {
     public containerId: string | null = null;
 
     async startContainer(): Promise<void> {
         try {
-            // Start the TEE simulator container
-            const { stdout } = await execAsync(
-                `docker run -d --rm -p ${TEE_SERVER_PORT}:${TEE_SERVER_PORT} phalanetwork/tappd-simulator:latest`
-            );
-            this.containerId = stdout.trim();
+            const LOCAL_COMPOSE_FILE_PATH = path.join(process.cwd(), "..", ".tee-cloud/compose-files/tee-compose.yaml");
+            if (IS_DEV) {
+                // Development: Use docker-compose
+                await execAsync(
+                    `DSTACK_ENDPOINT=http://localhost:8090 docker-compose -f ${LOCAL_COMPOSE_FILE_PATH} up -d`
+                );
+                const { stdout } = await execAsync(`docker-compose -f ${LOCAL_COMPOSE_FILE_PATH} up -d`);
+                this.containerId = stdout.trim();
+            } else {
+                // Production: Use direct docker command
+                const { stdout } = await execAsync(
+                    `docker run -d --rm -p ${TEE_SERVER_PORT}:${TEE_SERVER_PORT} jonathanpaella/agentlaunchpadstarterkit:latest`
```

Comment:
this line seems incorrect, we need to run something that deploys the image in phala cloud. see if you can find those lines, else leave this comment open

Reply:
also we need to obtain the url of the server somehow